docs/git/pull-request-guidelines.md

```diff
@@ -7,32 +7,23 @@ title: Pull Request Guidelines
 # Generic Pull Request Guidelines
 
 
-## Creator Guideline
-
-
-
-* Reflect the story/task/ in JIRA to GIT - Description, Link, Design reference, Any additional info.
-    * Think simple
-    * Compose clean, elegant, and readable code.
-* Document only meta information inadequately represented by code
-    * Focus on the Statement of user value
-* Create a branch equivalent to the JIRA ticket - PROJECT-XXX
-* Push your code
-    * ASAP - As soon as possible
-    * AFAP - As frequently as possible
-    * The goal here is to collaborate well with the latest changes. Build once and use many times.
-* Requesting for appropriate reviewers. Follow the steps as
-    * Drafting pull requests
-    * Review by self
-    * Review by others
-* Timely acknowledge feedback on the pull requests
-* Be patient about pull requests
-* Not parallelize too much and create multiple pull requests
-    * In-progress limit of at max 2 pull requests
-* Proper closure and definition of done
-* Open Mindset
-    * Acceptive
-    * Open to enhancements/improvements
+## Consider the following points while submitting and reviewing a pull request.
```

Comment:
Can we add a point which shows if the functionality/requirement is fully solved by the code changes or not?

Reply:
"Review for Functionality" is included. That should do it.